### PR TITLE
Added 'Moonlight Village' dungeon and updated some reactions

### PR DESCRIPTION
**Pull request by sjkd23**
Tested it and it seems to work!

### CODE REVIEW

**Review by ewang2002 — CHANGES_REQUESTED** [review]
One minor change needed. Also, describe what changes you made with the reactions.

Looks good otherwise.

File: src/constants/dungeons/MappedAfkCheckReactions.ts [hunk 1]
```diff
@@ -640,6 +640,15 @@ export const MAPPED_AFK_CHECK_REACTIONS: IMappedAfkCheckReactions = {
         name: "Mad God Mayhem Key",
         isExaltKey: false
     },
+    MV_KEY: {
```
[the remark this answers is not in the document]
Comment (sjkd23) [reply src/constants/dungeons/MappedAfkCheckReactions.ts:643]: fair

**Comment by sjkd23** [comment]
The updates to reactions was just removing any instance of 'Fungal Tome' receiving early loc and making sure it was formatted correctly after the changes I made.

**Review by ewang2002 — APPROVED** [review]

**Comment by ewang2002** [comment]
@nyapat need your approval to merge.

**Pull request merged**